.ci/infra/terraform/main.tf

```diff
@@ -314,6 +314,7 @@ module "project-services" {
     "securesourcemanager.googleapis.com",
     "securetoken.googleapis.com",
     "securitycenter.googleapis.com",
+    "securitycentermanagement.googleapis.com",
```

Comment:
Can you please put this change in a separate PR? It prevents the checks to be run in this PR.

Reply:
Sure, I will take it out from here.